src/jit.h

```diff
@@ -56,20 +57,14 @@ typedef void (*exec_t2c_func_t)(riscv_t *);
 #endif
 
 #if RV32_HAS(JIT_CACHE)
-#if !defined(__x86_64__) && !defined(__aarch64__)
-/* make sure LLVM can access the correct address of structure member */
-#error "The JIT-cache only supports 64-bit machine (LP64)."
-#endif
-
 #define MAX_JIT_CACHE_TABLE_ENTRIES (1 << 12)
 
 /* clang-format off */
```

Comment:
Drop clang-format off/on pair.

Reply:
The format will become a little weird, like
```c
struct jit_cache {
    __ALIGNED(4) uint32_t pc; /* program counter of ELF */
    __ALIGNED(4)
    uint32_t from; /* from whether t1c or t2c, ensure the correct
                      alignment for accessing member in LLVM IR
                    */
    void *entry;   /* entry of JIT-ed code */
};
```
Should we still remove it?

Reply:
Rewrite as `uint32_t pc __ALIGNED(4);`